ui/app/AppLayouts/Chat/panels/communities/CommunityWelcomeBannerPanel.qml

```diff
@@ -13,31 +11,22 @@ import utils 1.0
 
 Rectangle {
     id: root
-    height: 220
-    anchors.left: parent.left
-    anchors.leftMargin: Style.current.padding
-    anchors.right: parent.right
-    anchors.rightMargin: Style.current.padding
-    border.color: Style.current.border
-    radius: 16
-    color: Style.current.transparent
+
     property var activeCommunity
     property var store
     property var communitySectionModule
     property bool hasAddedContacts
 
     signal manageCommunityClicked()
 
-    MouseArea {
-        anchors.fill: parent
-        acceptedButtons: Qt.RightButton
-        propagateComposedEvents: true
-        onClicked: {
-            /* Prevents sending events to the component beneath
-               if Right Mouse Button is clicked. */
-            mouse.accepted = false;
-        }
-    }
+    height: childrenRect.height + Style.current.padding
+    anchors.left: parent.left
```

Comment:
It would be nice to remove anchoring from this place and use in place where the component is used.

Reply:
I agree; these 3 panels need some refactoring, it's 80% duplicate code. I just didn't want to spend more time on this than necessary